packages/jest-leak-detector/package.json

```diff
@@ -14,8 +14,7 @@
     "pretty-format": "^24.8.0"
   },
   "devDependencies": {
-    "@types/weak": "^1.0.0",
```

Comment:
we'll need typings for `weak-napi` - it'd br great to either push it to `weak-napi`, DT or just have them live in `jest-leak-detector`. You can just copy `weak`'s ones from DT - it's the same interface barring `isNearDeath` should always be `false`, not `boolean`

Reply:
I created a PR at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36869 to add type for `weak-api` package, I will update this PR if PR in DT has merged.